frontend/src/components/pagination-tests.cy.js

```diff
@@ -0,0 +1,80 @@
+import Pagination from './Pagination'
+
+describe('<Pagination />', () => {
+  let defaultProps
+
+  beforeEach(() => {
+    defaultProps = {
+      onClickPrev: cy.stub(),
+      onClickNext: cy.stub(),
+      onChange: cy.stub(),
+    }
+  })
+
+  it('Shows "Page 1 of "n"" (where "n" is the number of pages)', () => {
+    const props = {
+      currentPage: 1,
+      paginationInfo: {
+        totalPages: 5,
+        limit: 10
+      },
+      ...defaultProps,
+    }
+    cy.mount(<Pagination {...props} />)
```

Comment:
```suggestion
    cy.mount(<Pagination {...props} />)

```

Recomendo deixar uma linha em branco entre pré-condição, ação e resultado esperado.

Tal prática facilita a leitura e visualização de cada parte do teste.

A propósito, recomendo a leitura do conteúdo [_Anatomy of a Test_](https://www.epicweb.dev/anatomy-of-a-test).